src/Initialization/init_ModelingToolkit.jl

```diff
@@ -21,22 +22,24 @@ eval(quote
 
    ```julia
    julia> vars = @variables x[1:2] y
-   (Operation[x₁, x₂], y)
+   (Num[x₁, x₂], y)
 
    julia> LazySets._vec(vars)
-   3-element Array{Operation,1}:
+   3-element Array{Num,1}:
     x₁
     x₂
     y
    ```
    """
-    function _vec(vars::NTuple{L, Union{<:Operation, <:Vector{Operation}}}) where {L}
+    function _vec(vars::NTuple{L, Union{<:Num, <:Vector{Num}}}) where {L}
         return collect(reduce(vcat, vars))
     end
 
     # case with a single variable
-    _vec(vars::Tuple{Operation}) = [vars[1]]
+    _vec(vars::Tuple{Num}) = [vars[1]]
 
+    _get_variables(expr::Term) = ModelingToolkit.get_variables(expr)
+    _get_variables(expr::Vector{<:Term}) = unique(reduce(vcat, _get_variables(ex) for ex in expr))
```

Comment:
Tip (thanks to @isaacsas in [this comment](https://github.com/SciML/ModelingToolkit.jl/issues/731#issuecomment-758990856)): use the in-place version `ModelingToolkit.get_variables!` to accumulate the unique variables on the same array.

Reply:
So do you want to change it?

Reply:
So do you want to change it?

Reply:
You shouldn't dispatch on `Term`. You need to check a expression by testing `istree(expr)`.